src/main.js

```diff
@@ -51,7 +50,7 @@ function saveToArray() {
   currentCard.saveToStorage()
   updateCardDisplay();
   clearInput();
-  disableSaveButton()
+  disableSaveButton();
```

Comment:
Good catch on this.  In the future, you will be using a linter that can catch these types of errors.